app/views/admin/enterprise_roles/_enterprise_role.html.haml

```diff
@@ -1,5 +1,5 @@
 %td {{ enterprise_role.user_email }}
-%td= t('admin.enterprise_roles.manages')
+%td= t('.manages')
```

Comment:
first time I see this. Does Rails infer the namespace from the file path? :astonished: